source/_integrations/bring.markdown

```
@@ -26,12 +26,36 @@ related:
+- **Automated notifications:** Receive alerts on your phone when essential items are added to your list or when the quantity of items reaches a set value.
+- **List updates based on events:** Automatically add items to your shopping list when appliances are low on supplies, like dishwasher salt, or need routine maintenance, such as tub cleaner for the washer.
+- **Voice control:** Use voice assistants connected to Home Assistant to add items to your **Bring!** list.
+- **Geo-fencing:** Receive reminders when you are near a specific store and need to pick up items, based on your location.
+
+## Prerequisites
+
+For authentication, the integration requires the `email` and `password` of your **Bring!** account.  If you don’t have an account, you can easily sign up via the mobile app or WebApp.
 
 {% include integrations/config_flow.md %}
 
+#### Configuration parameters
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)